### PR TITLE
Add microtype and Latin Modern font

### PR DESCRIPTION
**Pull request by koppor**
This PR adds

- [microtype](http://texdoc.net/pkg/microtype) to have better looking pagraphs.
- [cfr-lm](http://texdoc.net/pkg/cfr-lm) which loads latin modern, which is the "modern" variant of the default font computer modern. See https://tex.stackexchange.com/a/247543/9075 for more information on cfr-lm

### CODE REVIEW

**Comment by Strauman** [comment]
I don't know what all of these options do, so I'll look into it. If it checks out I'll add it to the package :) 👍

File: src/packages.tex [hunk 1]
```diff
@@ -6,9 +6,29 @@
...
-% Mattematikk
+% Mattematik
 \usepackage{amsmath}
 \usepackage{mathtools}
 \usepackage{bm}
 \usepackage{esint}
 \usepackage{iflang}
+
+% EN: Character protrusion and font expansion. See https://www.ctan.org/pkg/microtype
+% DE: Optischer Randausgleich und Grauwerktkorrektur
```
Comment (Strauman) [inline src/packages.tex]: Line 17-20: Can you write these comments in English, please?

Comment (Strauman) [inline src/packages.tex]: Line 17-20: Can you write these comments in English, please?

**Comment by koppor** [comment]
Thank you for the hint. The German comments were from around 2008 and are outdated now - we are a decade later now :)

I fixed the comments also in the `config.tex` in the [scientific thesis template](http://latextemplates.github.io/scientific-thesis-template/): https://github.com/latextemplates/scientific-thesis-template/commit/41f52f459f571869152e935d3937eac37ea5d338

**Comment by Strauman** [comment]
I hope it's okay that I add a link to your github user in the docs as a thanks?

**Comment by koppor** [comment]
Sure! Thank you! 🎉

**Pull request merged**